hazelcast/src/main/java/com/hazelcast/config/AbstractXmlConfigHelper.java

```diff
@@ -288,10 +289,7 @@ public static String cleanNodeName(final Node node) {
     }
 
     protected static boolean getBooleanValue(final String value) {
-        String lowerCaseValue = StringUtil.lowerCaseInternal(value);
-        return "true".equals(lowerCaseValue)
-                || "yes".equals(lowerCaseValue)
-                || "on".equals(lowerCaseValue);
+        return parseBoolean(value);
```

Comment:
the old implementation always uses `Locale.ENGLISH` for a conversion to lower-case. 

The new one will use whatever is platform default. Can't this lead to issues with some locales? I know Turkish locale has somewhat unusual rules for conversion between i / I.